framework/src/systems/NonlinearSystemBase.C

```diff
@@ -168,7 +168,7 @@ NonlinearSystemBase::NonlinearSystemBase(FEProblemBase & fe_problem,
     _compute_jacobian_blocks_timer(registerTimedSection("computeJacobianBlocks", 3)),
     _compute_dampers_timer(registerTimedSection("computeDampers", 3)),
     _compute_dirac_timer(registerTimedSection("computeDirac", 3)),
-    _compute_scaling_jacobian_timer(registerTimedSection("computeScalingJacobian", 2)),
+    _compute_scaling_timer(registerTimedSection("computeScaling", 2)),
```

Comment:
Make sure you initialize your new class member Boolean here as well just in case we refactor and it doesn't get set later.

Reply:
Do you mean the new `Real _resid_vs_scaling_param` class member. I don't think I added any new uninitialized booleans...